pkg/config/applications/application.go

```diff
@@ -0,0 +1,550 @@
+package applications
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"strconv"
+	"strings"
+
+	"github.com/ibm-verify/verify-sdk-go/internal/openapi"
+	contextx "github.com/ibm-verify/verify-sdk-go/pkg/core/context"
+	errorsx "github.com/ibm-verify/verify-sdk-go/pkg/core/errors"
+)
+
+type ApplicationListResponse struct {
+	Embedded   Embedded `json:"_embedded" yaml:"_embedded"`
+	TotalCount int      `json:"totalCount" yaml:"totalCount"`
+}
+
+type Embedded struct {
+	Applications []*Application `json:"applications" yaml:"applications"`
+}
+
+type Application struct {
+	Name                   string                 `json:"name" yaml:"name"`
+	TemplateID             string                 `json:"templateId" yaml:"templateId"`
+	Links                  Links                  `json:"_links" yaml:"_links"`
+	Providers              Providers              `json:"providers" yaml:"providers"`
+	Provisioning           Provisioning           `json:"provisioning" yaml:"provisioning"`
+	AttributeMappings      []*AttributeMapping    `json:"attributeMappings" yaml:"attributeMappings,omitempty"`
+	ApplicationState       bool                   `json:"applicationState" yaml:"applicationState,omitempty"`
+	ApprovalRequired       bool                   `json:"approvalRequired" yaml:"approvalRequired,omitempty"`
+	SignonState            bool                   `json:"signonState" yaml:"signonState,omitempty"`
+	Description            string                 `json:"description" yaml:"description,omitempty"`
+	ProvisioningMode       string                 `json:"provisioningMode" yaml:"provisioningMode,omitempty"`
+	IdentitySources        []string               `json:"identitySources" yaml:"identitySources,omitempty"`
+	VisibleOnLaunchpad     bool                   `json:"visibleOnLaunchpad" yaml:"visibleOnLaunchpad,omitempty"`
+	Customization          Customization          `json:"customization" yaml:"customization,omitempty"`
+	DevportalSettings      DevportalSettings      `json:"devportalSettings" yaml:"devportalSettings,omitempty"`
+	APIAccessClients       []*APIAccessClients    `json:"apiAccessClients" yaml:"apiAccessClients,omitempty"`
```

Comment:
I believe this represents an array of clients. So the data type should be `[]*APIAccessClient`?